sdk/include/ndk/section_attribs.h

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Should this have the standard NDK header?

Reply:
Fixed